grammars.yml

```diff
@@ -1057,6 +1057,11 @@ vendor/grammars/verilog.tmbundle:
 - source.verilog
 vendor/grammars/vhdl:
 - source.vhdl
+vendor/grammars/vsc-ember-syntax:
+- inline.hbs
+- inline.template
+- source.gjs
+- source.gts
```

Comment:
@gilest very happy to see this PR! I hope it'll get accepted as soon as possible.

For the time being, I fallback to `JS` or `TS` when I want to have a usable syntax highlighting on github.com for my `.gjs` and `.gts` files

If this PR is only for "Glimmer JS", is this line supposed to be here?

Reply:
I think so, yes. Because when Glimmer TS is added it will use the same grammar

Those entries are generated when running `script/add-grammar`